inductiva/client/apis/paths/gcp_instances_storage_dir_name.py

```diff
@@ -0,0 +1,6 @@
+from inductiva.client.paths.gcp_instances_storage_dir_name.delete import ApiFordelete
```

Comment:
I don't understand this name convention?
Should this be APIForDelete?
Also, are we still importing classes?
Maybe this is auto-generated because it is the client? Is that it?

@hpenedones @begaiym-k @luispcunha : can you please clarify?

Reply:
Yes, it is the generated client.

Reply:
OK! But, just out of curiosity: would we be able to have the generator generate:

1) better names?
2) shorter structures. For example, do we need all this:

```
inductiva.client.paths.gcp_instances_storage_dir_name.delete
```

I am not saying this is a priority, but I want to make sure we are not "leaving crumbs"...

@hpenedones @begaiym-k @luispcunha : can you please clarify?

Reply:
ok! thank you for the explanation.
I am assuming the InstanceApi is also that way because of the autogeneration (i.e. no InstaceAPI)?